.github/workflows/reuse-tests-7to8.yml

```diff
@@ -13,7 +13,7 @@ on:
         type: string
       tmt_plan_regex:
         type: string
-        default: "^(?!.*c2r)(?!.*sap)(?!.*8to9)"
+        default: "^(?!.*upgrade_plugin)(?!.*c2r)(?!.*sap)(?!.*8to9)"
```

Comment:
Should this also include the `(?!.*tier[2-3].*)(?!.*rhsm)` part?

Reply:
Fixed, thanks